server.ts

```diff
@@ -13,24 +13,59 @@ config(); //Read .env file lines as though they were env vars.
 // { rejectUnauthorized: false } - when connecting to a heroku DB
```

Comment:
It would be nice to see SQL queries for creating and populating database structure so that we can see how the SQL queries here link across to and access the database